src/client/dfuse/ops/write.c

```diff
@@ -24,18 +24,49 @@
 #include "dfuse_common.h"
 #include "dfuse.h"
 
+static void
+dfuse_cb_write_complete(struct dfuse_event *ev)
+{
+	if (ev->de_ev.ev_error == 0)
+		DFUSE_REPLY_WRITE(ev, ev->de_req, ev->de_len);
+	else
+		DFUSE_REPLY_ERR_RAW(ev, ev->de_req,
+				    daos_der2errno(ev->de_ev.ev_error));
```

Comment:
Is this errno already coverted?

Reply:
no i think it will be a DER. so what you have here is OK.